build/cqs.tsv

```diff
@@ -74,6 +76,7 @@ net.java.dev.jsr-275:jsr-275	1.0-beta-2	compile
 net.minidev:asm	1.0.2	compile
 net.minidev:json-smart	2.1.1	compile
 net.sf.geographiclib:GeographicLib-Java	1.44	compile
+org.antlr:antlr4-runtime	4.5	compile
```

Comment:
@elahrvivaz, @jahhulbert-ccri @mdzimmerman good news!  We have CQs for these 3 additions.  Antlr and Concurrent trees are completely approved, and we have parallel check-in for CQEngine.

That means that we can :shipit: when ready.